UPGRADE-2.0.md

```diff
@@ -180,6 +180,15 @@
       `Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository` to
       `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAssociationRepository`
 
+* The following helper classes and interfaces have been removed:
```

Comment:
If this list point is general purpose removal mention, then it should be like so,
```suggestion
* The following classes and interfaces have been removed:
```

Reply:
For now it is only for helpers so I leave it as it is, in the nearest future we will definitely go through all the changes in 2.0 and update this file 🖖🏻